src/appkit.cs

```diff
@@ -17906,7 +17996,7 @@ interface NSTextAttachmentCell {
 	}
 
 	[BaseType (typeof (NSObject))]
-	interface NSTextAttachment : NSCoding {
+	interface NSTextAttachment : NSCoding, NSSecureCoding {
```

Comment:
minor: `NSSecureCoding` is a subclass of `NSCoding` so you don't really need to keep the later